atd-vze/src/views/Crashes/recommendationsDataMap.js

```diff
@@ -0,0 +1,20 @@
+export const recommendationsDataMap = [
+    {
+      title: "Fatality Review Board Recommendations",
+  
+      fields: {
+        coordination_partner_id: {
+          label: "Coordination Partner: ",
+          lookupOptions: "atd__coordination_partners_lkp",
+        },
+        recommendation_status_id: {
+          label: "Status: ",
+          lookupOptions: "atd__recommendation_status_lkp"
```

Comment:
i notice you're not using `lookupOptions`. if you wanted to, you might rewrite `displayData` like this:

```javascript
  const displayData = (data, { lookupOptions, key }) => {
    if (lookupOptions) {
      return data[lookupOptions][key];
    } else {
      return data[key];
    }
  };
```
and add the key you want to access to you config

```javascript
export const recommendationsDataMap = {
  title: "Fatality Review Board Recommendations",

  fields: {
    coordination_partner_id: {
      label: "Coordination Partner: ",
      lookupOptions: "atd__coordination_partners_lkp",
      key: "description",
    },
    recommendation_status_id: {
      label: "Status: ",
      lookupOptions: "atd__recommendation_status_lkp",
      key: "description",
    },
    text: {
      label: "Note",
      key: "text",
    },
  },
};
```

and finally, the JSX

```javascript
 <td style={{width: "65%"}}>
      <b>{fieldConfig.fields.recommendation_status_id.label}</b>
      {displayData(recommendation, fieldConfig.fields.recommendation_status_id)}
  </td>
```

☝️ i haven't tested this, and what you have is already fine, but it gets at the idea of letting your config help you write less code, and centralizing as much of the hard-coded stuff as possible in one place (your config).

Reply:
okay this answers my previous question about whether a data map is necessary. i like this, its a looot cleaner. thank you!!

Reply:
yep, and if you had many more columns to deal with, that's when you'd want to start thinking about a custom table cell component, something like

```javascript
```